src/Alert/index.jsx

```diff
@@ -1,40 +1,108 @@
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-import Alert from 'react-bootstrap/Alert';
+import classNames from 'classnames';
+import BaseAlert from 'react-bootstrap/Alert';
+import { useMediaQuery } from 'react-responsive';
 import { Icon } from '..';
+import Button from '../Button';
+import { breakpoints } from '../Responsive';
+import ActionRow from '../ActionRow';
 
-const WrappedAlert = React.forwardRef(({
+const Alert = React.forwardRef(({
   children,
   icon,
+  actions,
+  dismissible,
+  onClose,
+  closeLabel,
+  stacked,
   ...props
-}, ref) => (
-  <Alert
-    {...props}
-    className={`alert-content ${props.className}`}
-    ref={ref}
-  >
-    {icon && <Icon src={icon} className="alert-icon" />}
-    <div className="alert-message-content">
-      {children}
-    </div>
-  </Alert>
-));
-
-WrappedAlert.propTypes = {
-  ...Alert.propTypes,
+}, ref) => {
+  const [isStacked, setIsStacked] = useState(stacked);
+  const isExtraSmall = useMediaQuery({ maxWidth: breakpoints.extraSmall.maxWidth });
+  const actionButtonSize = 'sm';
+
+  useEffect(() => {
+    if (isExtraSmall) {
+      setIsStacked(true);
+    } else {
+      setIsStacked(stacked);
+    }
+  }, [isExtraSmall, stacked]);
+
+  const cloneActionElement = useCallback(
+    (Action) => {
+      const addtlActionProps = { size: actionButtonSize, key: Action.props.children };
+      return React.cloneElement(Action, addtlActionProps);
+    },
+    [],
+  );
+
+  return (
+    <BaseAlert
+      {...props}
+      className={classNames('alert-content', props.className)}
+      ref={ref}
+    >
+      {icon && <Icon src={icon} className="alert-icon" />}
+      <ActionRow
+        isStacked={isStacked}
+        isStackedReversed={false}
+        isStackedCentered={false}
+      >
```

Comment:
Looks like you're using action row in a different way than I expected.

I thought we should use ActionRow as a replacement for the pgn_alert-actions div.